core/controllers/base_test.py

```diff
@@ -831,8 +831,8 @@ def test_responses_with_valid_iframe_restriction(self):
         self.get_html_response('/mock')
 
         response = self.get_html_response(
-            '/mock', params={'iframe_restriction': 'DENY'})
-        self.assertEqual(response.headers['X-Frame-Options'], 'DENY')
+            '/mock', params={str('iframe_restriction'): str('DENY')})
```

Comment:
Why these changes?

Reply:
Hi @apb7, This is also for a similar reason I mentioned above